fbw-a32nx/src/systems/instruments/src/RMP/Components/BaseRadioPanels.tsx

```diff
@@ -93,46 +108,46 @@ const PoweredRadioPanel = (props: Props) => {
   useInteractionEvent(`A32NX_RMP_${props.side}_VOR_BUTTON_PRESSED`, () => {
     if (navButtonPressed) {
       setPanelMode(6);
-      setNavTransceiverType(TransceiverType.VOR);
+      setNavReceiverType(TransceiverType.VOR);
     }
   });
 
   useInteractionEvent(`A32NX_RMP_${props.side}_ILS_BUTTON_PRESSED`, () => {
     if (navButtonPressed) {
       setPanelMode(7);
-      setNavTransceiverType(TransceiverType.ILS);
-    }
-  });
-
-  useInteractionEvent(`A32NX_RMP_${props.side}_GLS_BUTTON_PRESSED`, () => {
-    if (navButtonPressed) {
-      setPanelMode(8);
-      setNavTransceiverType(TransceiverType.GLS);
+      setNavReceiverType(TransceiverType.ILS);
     }
   });
 
+  /**
+   * MLS IMPLEMENTED IN THE XML BEHAVIOURS
+   * BUT DISABLED HERE SINCE THERE IS NOT ENOUGH REFERENCES
```

Comment:
We will never implement MLS as it's obsolete.

Reply:
True! Was comment out but I suppose the rebase wasn't that successful. It's fixed.

Also, I'm keeping MLS tag since the it's written MLS everywhere in the code but it should be GLS in the future. Will it be implemented?

Reply:
The GLS button is already there alongside MLS in the cockpit. GLS will be implemented in time with a new MMR implementation. I will be a bit busy for the next couple of months though.